sc_server_auth/create_admin.py

```diff
@@ -0,0 +1,53 @@
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from getpass import getpass
+
+from sc_server_auth.configs.log import get_default_logger
+from sc_server_auth.configs.models import Database
+from sc_server_auth.configs.parser import get_config
+from sc_server_auth.server.admin import verify_user_info_in_database
+from sc_server_auth.server.database import DataBase
```

Comment:
I think, it's not good to use same names (database), especially if they differ by letter

Reply:
You're right, I renamed Database on DatabaseType